src/RPCServer.ts

```diff
@@ -81,12 +81,12 @@ class RPCServer {
    * transform streams that convert `Uint8Array` to `JSONRPCRequest` on the forward
    * path and `JSONRPCResponse` to `Uint8Array` on the reverse path.
    * @param obj.streamKeepAliveTimeoutTime - Time before a connection is cleaned up due to no activity. This is the
-   * value used if the handler doesn't specify its own timeout time. This timeout is advisory and only results in a
+   * value used if the handler doesn't specify its own Handler.timeout time. This Handler.timeout is advisory and only results in a
    * signal sent to the handler. Stream is forced to end after the timeoutForceCloseTime. Defaults to 60,000
    * milliseconds.
-   * @param obj.timeoutForceCloseTime - Time before the stream is forced to end after the initial timeout time.
-   * The stream will be forced to close after this amount of time after the initial timeout. This is a grace period for
-   * the handler to handle timeout before it is forced to end. Defaults to 2,000 milliseconds.
+   * @param obj.timeoutForceCloseTime - Time before the stream is forced to end after the initial Handler.timeout time.
+   * The stream will be forced to close after this amount of time after the initial Handler.timeout. This is a grace period for
+   * the handler to handle Handler.timeout before it is forced to end. Defaults to 2,000 milliseconds.
```

Comment:
This docblock needs to be updated, the names of the parameters are out of sync and we don't even have a force timeout anymore.

Make sure the parameter descriptions align with the parameters.